test/src/test/java/hudson/UDPBroadcastThreadTest.java

```diff
@@ -55,7 +55,8 @@ private static void updatePort(int newValue) throws Exception {
      * Old unicast based clients should still be able to receive some reply,
      * as we haven't changed the port.
      */
-    @Test public void legacy() throws Exception {
```

Comment:
I think we should use the standard `@Ignore` out is there a compelling reason not to use this?